### PR TITLE
general text styling close #74

### LINKED ISSUE #74: Rediseño de sitio web / cambiar fuente por web-safe
Cambiar la fuente descargada por una fuente que se pueda utilizar desde el navegador propio

tentativa: Arial, Helvetica, sans-serif

### PR DESCRIPTION
**Pull request by givanse**

### CODE REVIEW

File: assets/_scss/_variables.scss [hunk 1]
```diff
@@ -6,7 +6,10 @@ $smx-pink-color: #F3476D;
...
 $smx-black-color: #000;
 
-$smx-font-family: 'Roboto Mono', sans-serif;
+$smx-font-family: sans-serif, arial, helvetica;
+
 $smx-font-bolder: 700;
+$smx-font-bold: 600;
 $smx-font-regular: 400;
 $smx-font-lighter: 300;
+$smx-font-lightest: 100;
```
Comment (givanse) [inline assets/_scss/_variables.scss:15]: variables names suggestions welcome

**Comment by xiam** [comment]
Me parece bien, solo hay un pequeño conflicto @givanse

**Comment by givanse** [comment]
rebased

**Comment by gabelula** [comment]
@givanse podes chequear con este PR https://github.com/CodeandoMexico/terremoto-cdmx/pull/83 ?

**Comment by givanse** [comment]
closing, too many cooks in this kitchen

**Pull request closed without merging**